SmartGameBooster.SmartGameBooster version 5.2.0

BajajAditya123 opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/27970)

Commit:
SmartGameBooster.SmartGameBooster version 5.2.0

BajajAditya123 commented:
@mstfbot Please hold this pull till someone approves it

BajajAditya123 commented:
@OfficialEsco  Please approve my changes

ItzLevvie commented on manifests/s/SmartGameBooster/SmartGameBooster/5.2.0/SmartGameBooster.SmartGameBooster.locale.en-US.yaml:
```diff
@@ -0,0 +1,13 @@
+Publisher: Smart Game Booster
+PackageName: Smart Game Booster
+License: © Smart Game Booster. All rights reserved.
+ShortDescription: It booosts FPS for better gaming experience. It protects your gaming accounts.
```
```suggestion
ShortDescription: It boosts FPS for better gaming experience. It protects your gaming accounts.
```

Commit:
Update SmartGameBooster.SmartGameBooster.locale.en-US.yaml

# Created using wingetcreate 0.4.0.3
# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json

PackageIdentifier: SmartGameBooster.SmartGameBooster
PackageVersion: 5.2.0
PackageLocale: en-US
Publisher: Smart Game Booster
PackageName: Smart Game Booster
License: © Smart Game Booster. All rights reserved.
ShortDescription: It boosts FPS for better gaming experience. It protects your gaming accounts.
ManifestType: defaultLocale
ManifestVersion: 1.0.0

BajajAditya123 commented:
Thank you for correcting me🙏🙏🙏

BajajAditya123 commented:
Can anyone review my changes??
